src/front-end/src/components/Login.js

```diff
@@ -1,7 +1,49 @@
-import React from 'react';
-
-const Login = () => {
-  return <div style={{ height: '500px' }}>Place login page here</div>;
-};
+import React, { Component } from 'react';
+import { Form, Button, Input } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
 
+// eslint-disable-next-line react/prefer-stateless-function
+class Login extends Component {
+  render() {
+    return (
+      <Form>
+        <p>Welcome to MentorMatch!</p>
+        <p>Login to access the mentorship portal</p>
+        <Form.Item
+          label="Email"
+          name="email"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>
```

Comment:
Can a 'type' field be added (type="email") so that the input is checked for valid format?

Reply:
added.